src/interfaces.ts

```diff
@@ -475,6 +476,44 @@ export interface FormikProperties {
   form: FormikHelpers<FormikValues>;
 }
 
+export interface Concept {
```

Comment:
Det er fleire PR i det siste som har lagt til interface for Concept. Må sjekke om det er overlapp på fleire av disse.